Spatial_searching/include/CGAL/Point_container.h

```diff
@@ -330,6 +422,32 @@ class Point_container {
     return b;
   }
 
+  template <class Separator>
+  void balanced_split(Point_container<Traits>& c, Separator& sep)
```

Comment:
Why is `sep` not `const`?

Reply:
Because we update this splitter inside this method.

Reply:
Where?

Reply:
Ooo, You are completely right. I had a look at the old version where the `balanced_split()` was a part of the `Splitters.h` and I have updated the `sep` inside. This one does not require it. I will fix it!